rpdc.py

```diff
@@ -19,7 +19,8 @@
 import jsonschema
 
 
-SchemaJSONPath = "schema/workflow_schema_v1_4.schema.json"
+# SchemaJSONPath = "schema/workflow_schema_v1_4.schema.json"
```

Comment:
Wouldn't have been necessary to keep the old one as a comment (I'll merge nonetheless and remove that line afterwards)